packages/timeline-state-resolver/src/integrations/httpWatcher/index.ts

```diff
@@ -138,16 +124,20 @@ export class HTTPWatcherDevice extends Device<DeviceOptionsHTTPWatcherInternal>
 		}
 	}
 
-	get canConnect(): boolean {
-		return false
-	}
 	get connected(): boolean {
 		return false
 	}
-	get deviceType() {
-		return DeviceType.HTTPWATCHER
+
+	convertTimelineStateToDeviceState(_state: Timeline.TimelineState<TSRTimelineContent>): HTTPWatcherDeviceState {
```

Comment:
No need to have these arguments if we aren't using them anyway and aren't likely to be later.